server/urls.py

```diff
@@ -17,6 +17,28 @@
    permission_classes=(permissions.AllowAny,),
 )
 
+
+
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
+
+# for testing
+schema_view = get_schema_view(
+   openapi.Info(
+      title="teesco-API",
+      default_version='v1',
+      description="Test description",
+      terms_of_service="https://www.google.com/policies/terms/",
+      contact=openapi.Contact(email="contact@snippets.local"),
+      license=openapi.License(name="BSD License"),
+   ),
+   public=True,
+   permission_classes=(permissions.AllowAny,),
+)
+
+
```

Comment:
```suggestion
```

The line:10-18 is already performing this job :innocent:

Reply:
okay sir